Shared/Realm.Shared/Handles/QueryHandle.cs

```diff
@@ -226,38 +226,38 @@ public void BinaryNotEqual(IntPtr columnIndex, IntPtr buffer, IntPtr bufferLengt
             nativeException.ThrowIfNecessary();
         }
 
-        public void StringContains(IntPtr columnIndex, string value)
+        public void StringContains(IntPtr columnIndex, string value, bool caseSensitive = true)
```

Comment:
Feels wrong to me to have default parameters in a class like this that is not a public API and where we probably want to be quite explicit..